src/migrate/MigrationGenerator.js

```diff
@@ -0,0 +1,97 @@
+import fs from 'fs';
+import path from 'path';
+import mkdirp from 'mkdirp';
+import Promise from 'bluebird';
```

Comment:
Can this be replaced with native Promise?

Reply:
When I did that, linting failed. Thoughts?

Reply:
There should be no need to focus on using native promises since knex already depends heavily on bluebird.

There's been suggestions before to not have this dependency - or at the very least the option to supply promise lib/native - but I believe until such a decision is made, it's fine to keep using bluebird.

Reply:
@wubzz 
https://github.com/tgriesser/knex/issues/1588 -> there is very clearly communicated intent to eventually go away off using bluebird, hence I see why not to avert using it in newly touched code.

@JakeGinnivan What was the linting error?

Reply:
@JakeGinnivan Ah, most likely linting error was due to use of Promise.promisify -> you can't just remove reference to Promise here, instead import promisify explicitly.

Reply:
@kibertoad Afaik that comment is strictly in regards to the "utility" functions exposed by knex _(such as `.map` etc)_ and TS typings, not removing bluebird as a whole. Especially not as a first step. 

In any case it's not on the board for the next version of knex and certainly not set in stone. Contributors should not have to take this **potential** future change into consideration when developing features in present time, imho.

Reply:
Linting error was Promise is not defined